### PR TITLE
Features/add new cost

### PR DESCRIPTION
**Pull request by Svetlanasyo**
Add expenses model, repositories and initial mocked data

### CODE REVIEW

File: src/main/java/net/svera/RestAPISvera/expenses/models/Expense.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    private String location;
+    @NonNull private String name;
+    @NonNull private Date date;
+    @NonNull  private double price;
+    private double quantity;
+    private String unit;
+    private Date expireDate;
+
+    @ManyToOne
+    @JoinColumn(name="subcategory_id", nullable = false)
```
Comment (TeXXaS) [inline src/main/java/net/svera/RestAPISvera/expenses/models/Expense.java:26]: Would it be better to have categories and subcategories in one table, as multi level? Here I have to have subcategory, while main category can't be used. What if there would be 3 levels?

File: src/main/java/net/svera/RestAPISvera/expenses/models/ExpenseCategory.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package net.svera.RestAPISvera.expenses.models;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.Set;
+
+
+@Data
```
Comment (TeXXaS) [inline src/main/java/net/svera/RestAPISvera/expenses/models/ExpenseCategory.java:9]: :+1:

File: src/main/java/net/svera/RestAPISvera/expenses/models/ExpenseCategory.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+@Entity
+@Table(name="EXPENSECATEGORY")
```
Comment (TeXXaS) [inline src/main/java/net/svera/RestAPISvera/expenses/models/ExpenseCategory.java]: Maybe `expense_category`?

Comment (Svetlanasyo) [reply src/main/java/net/svera/RestAPISvera/expenses/models/ExpenseCategory.java]: Changed

**Pull request merged**